app/Http/Controllers/ResultsController.php

```diff
@@ -55,6 +58,10 @@ public function index(MismatchGetRequest $request, WikibaseAPIClient $wikidata):
             $mismatches->isNotEmpty() ? [ 'results' => $mismatches->groupBy('item_id') ] : []
         );
 
+        //collect metric
+        Log::info("statsd 'mismatch_request' (via web)");
```

Comment:
If we want to merge this maybe we should remove the log? I can imagine the log getting cluttered really quickly once we being to use this in prod.

Reply:
yes, definitely. I introduced the logs for myself so that I can verify what's going on and then let them in there, so you can use them to understand what I changed. But I agree, we should definitely remove them again.